polaris.shopify.com/src/components/IconDetails/IconDetails.tsx

```diff
@@ -0,0 +1,145 @@
+import Link from "next/link";
+import { className } from "../../utils/various";
+import Image from "../Image";
+import CodeExample from "../CodeExample";
+import styles from "./IconDetails.module.scss";
+
+interface Props {
+  fileName: string;
+  iconData: {
+    name: string;
+    set: string;
+    description: string;
+    keywords: string[];
+  };
```

Comment:
Not this PRs fault, but we currently have 3 types for icons:

- `type Icon` in types.ts
- typeof iconsMetadata
- These types

Reply:
Fixed this up.